README.md

```diff
@@ -1,10 +1,78 @@
-# pallet-move
-MoveVM pallet for Substrate-based chains
+# Move Pallet
 
+A pallet for substrate based blockchains to enable the usage of smart contracts written in the Move-language.
 
-# Running pallet from the substrate-node
+## Overview
+
+Smart contracts can directly be implemented and executed as Move scripts or modularized in Move modules. Therefor, the pallet supports publishing of Move modules, and the execution of Move scripts to achieve this functionality. In case of larger projects, the pallet provides the publishing of a bundle (multiple Move modules).
```

Comment:
```suggestion
Smart contracts can directly be implemented and executed as Move scripts or modularized in Move modules. Therefore, the pallet supports publishing Move modules and the execution of Move scripts to achieve this functionality. In the case of larger projects, the pallet provides the publishing of a bundle (multiple Move modules).
```

Reply:
Mhh, not sure about the "Therefor", these are two different words... I think both can be used here, but I wanted to use that one... do you prefer "Therefore" in this case?

Rest is done

Reply:
Oops, sorry! - my Grammarly reported that one. Feel free to ignore my comment and resolve this discussion